### ISSUE TITLE
Adding stoichiometric compound  functions

### ISSUE THREAD

**Issue by wahab2604** [PhasesResearchLab/ESPEI#131]
I understand ESPEI fits stoichiometric compounds from the unary functions and the experimental data in the JSON input files, but if you have a stoichiometric gibbs function from literature, could you add it into the phase diagram fitting without the need for ESPEI to fit it initially

Thanks,

**Comment by richardotis** [comment]
ESPEI can use any TDB file as input, so you could hard-code the function or phase of interest in the TDB file, and then pass it to ESPEI as input for the MCMC runs.

**Comment by wahab2604** [comment]
Thank you, If I wish to use an associates model for the liquid phase in the TDB file that I input to ESPEI, how should I define this in the phase models JSON or anywhere else ?

**Comment by bocklund** [comment]
The phase names and constituents in the phase models file are al read from the TDB, so nothing special needs to be done if you have models in the TDB that are compatible with pycalphad's `Model` class (which is all the standard TDB stuff including the CEF, partitioned models, ionic liquids, and associate models). For user-implemented models, they can be specified as implemented by #202:

```
{
  "components": ["CU", "MG", "VA"],
  "phases": {
         "LIQUID" : {
            "sublattice_model": [["CU", "MG"]],
            "sublattice_site_ratios": [1],
            "model": "mypackage.mymodule.MyModel"
         },
         "FCC_A1": {
            "sublattice_model": [["CU", "MG"], ["VA"]],
            "sublattice_site_ratios": [1, 1]
         }
    }
}
```

where `mypackage.mymodule.MyModel` is the fully qualified import path for a user-defined `MyModel`.
